### DISCUSSION TITLE
Better Spectator

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#768] [__Dismissed__] [answered]
a beter spectator option that if u go in a players view in spectator and he dies/teleports u wont  get kicked out of his screen so u get teleported with him and if he respawns u automatically get back in his screen. would be cool for pvp/survival servers where moderators are spectating players because of hacking/xray and they always have to teleport with them

**Comment by BillyGalbreath** [comment, marked as answer]
This is outside server control. I can teleport the spectator back to the player, but the client wont "connect" to them.
